CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+1.5.3 Release notes (2021-11-12)
```

Comment:
może warto 1.6.0 bo zmieniasz jednak metodę publiczną to jak im to wpadnie automatem to build sie sypnie